Sofa/Component/MechanicalLoad/src/sofa/component/mechanicalload/DiagonalVelocityDampingForceField.inl

```diff
@@ -30,16 +30,18 @@ namespace sofa::component::mechanicalload
 
 template<class DataTypes>
 DiagonalVelocityDampingForceField<DataTypes>::DiagonalVelocityDampingForceField()
-    : dampingCoefficients(initData(&dampingCoefficients, "dampingCoefficient", "velocity damping coefficients (by cinematic dof)"))
+    : d_dampingCoefficients(initData(&d_dampingCoefficients, "dampingCoefficient", "velocity damping coefficients (by cinematic dof)"))
 {
+    //SOFA_ATTRIBUTE_DEPRECATED("v23.12", "v24.06", "This Data is now replaced by d_dampingCoefficients")
+    dampingCoefficients.setParent(d_dampingCoefficients);
```

Comment:
```suggestion
    dampingCoefficients.setParent(&d_dampingCoefficients);
```

The compilation fails because the prototype of `setParent` is:
```cpp
bool setParent(BaseData* parent, const std::string& path = std::string());
```